docs/caching/includes/redis-intro.md

```diff
@@ -0,0 +1,5 @@
+---
+ms.topic: include
+---
+
+[Redis](https://redis.io/) is the world's fastest data platform for caching, vector search, and NoSQL databases. The .NET Aspire Redis integration enables you to connect to existing Redis instances, or create new instances from .NET with the [`docker.io/library/redis` container image](https://hub.docker.com/_/redis/).
```

Comment:
To me, this sounds like a marketing statement that might be proven, especially from a docs standpoint. I would just reword that to be less absolute. https://azure.microsoft.com/en-us/products/cache/ has some statements you might be able to use without saying specific that it's the "world's fastest."

> Superior throughput and performance to handle millions of requests per second with down to sub-millisecond latency

> Add a quick caching layer to the application architecture to handle thousands of simultaneous users with near-instant speed—all with the benefits of a fully managed service.

Reply:
It's intentional, from their marketing page.